doc/source/changes/version_0_30.rst.inc

```diff
@@ -0,0 +1,32 @@
+﻿Syntax changes
+--------------
```

Comment:
I remember that you changed the template for the release notes in a (very) recent PR. You may want to use the new template?